clean

```diff
@@ -5,7 +5,7 @@ cd $(dirname "$0")
 echo "Cleaning up all intermediate and compiled files generated from Cython compilation..."
 
 rm -rf build dist
-rm -f $(find . -name "*.pyx" | sed s/pyx$/cpp/g)
+rm -f $(find . -name "*.pyx" | sed s/\.pyx$/\*\.cpp/g)
```

Comment:
This was because I ran `compile` and got `file_name2.cpp` at some pt, but I can remove it.